core/templates/domain/improvements/high-bounce-rate-task.model.spec.ts

```diff
@@ -163,8 +163,8 @@ describe('High bounce rate task', function() {
             entity_id: 'eid',
             entity_version: 1,
             // This throws "Type '"???"' is not assignable to type
-            // '"high_bounce_rate"'." This is because 'task_type'
-            // should be equal to 'high_bounce_rate' but we set it
+            // '"high_bounce_rate"'.". We need to suppress this error because
```

Comment:
Ditto, why is the need of changing this?

Reply:
I changed this because it does not follow the pattern required for '@ts-ignore' or '@ts-expect-error' which I am implementing in comment-style check
https://github.com/oppia/oppia/pull/13022/files#diff-ebdbd754a4a10a16d3a71e77edcaa4e50310e3e7e14fbb47a456f5d0d0b4d272R92-R105